SourceryRuntime/Sources/Definable.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Describes that the object is defined in a context of some `Type`
+public protocol Defined: class {
```

Comment:
What does `Definition` protocol describe? I guess it describes that the object is a `definition` of something? I think `Defined` describes different behaviour.